### DISCUSSION TITLE
List out all the exceptions

### DISCUSSION THREAD

**Discussion by anitsh** [Giorgi/EntityFramework.Exceptions#55] [General]
It would be easy to find the list of the all the exceptions that are handled. It can be done either in the READEME or in a separate file EXCEPTIONS.md.

Or are these the list of exceptions being handled at the moment (as mentioned in the README) -  UniqueConstraintException, CannotInsertNullException, MaxLengthExceededException, NumericOverflowException, ReferenceConstraintException?

If so, then there is no further need a page 😃

**Comment by Giorgi** [comment]
Yes, the are the ones that are handled at the moment.
